src/core/search/scanner.h

```diff
@@ -29,6 +37,23 @@ class Scanner : public Lexer {
   dfly::search::location loc() {
     return location();
   }
+
+  Parser::symbol_type ParseParam(std::string_view name, const Parser::location_type& loc) {
+    name = name.substr(1);
```

Comment:
Do you want to `CHECK(!name.empty())`? Otherwise current code will throw an exception, and if you go with `remove_prefix()` it'd be undefined behavior

Reply:
True